### DISCUSSION TITLE
Select с интервалом в поле дата

### DISCUSSION THREAD

**Discussion by AaaronWalsh** [totumonline/totum-ru-issues-and-discussions#616] [Как это сделать?] [answered]
Перелистал документацию и не нашел упоминаний как сделать select по полю типа дата с указание интервала:
date between field1 and field2
Попытка указания через qrow дает ошибку If the right filter element is not a fieldName, then the left one must be
qrow`$date >='period_s' && $date <= 'period_po'`

**Comment by erpic-ru** [comment]
Тут не нужен qrow, пишете 2 where >= <=

**Reply by AaaronWalsh** [reply]
А если усложнить и надо два between через OR и т.п, а для where все условия должны быть true. Можно ли вообще задавать более сложные конструкции в qrow

**Reply by ttm-alexeypolunin** [reply, marked as answer]
В qrow name полей должны стоять в левой части:  name сравнение значение о чем и пишет ошибка выше

**Reply by ttm-alexeypolunin** [reply]
Сделай мне код с select который при помощи qrow выбирает строки где data_1 >= #start и <= #end или data_2 >= #limit_s и <= #limit_e

```totum
=: select(table: 'table_name'; field: 'field_name'; where: $qrow)
​
qrow: qrow`('data_1' >= #start && 'data_1' <= #end) || ('data_2' >= #limit_s && 'data_2' <= #limit_e)`
```

Наша модель говорит мне вот так попробовать, по идее должно сработать
